var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -0,0 +1,410 @@
+                ])
+
+            if '+cuda' in spec:
+                options.extend([
+                    '-DSEACASProj_ENABLE_Kokkos:BOOL=ON',
+                    '-DTPL_ENABLE_CUDA:Bool=ON',
+                    '-DCUDA_TOOLKIT_ROOT_DIR:PATH=${CUDA_PATH}',
+                    '-DKOKKOS_ENABLE_DEPRECATED_CODE:BOOL=OFF',
+                    '-DKokkos_ENABLE_Pthread:BOOL=OFF',
+                ])
+            else:
+                options.extend([
+                    '-DSEACASProj_ENABLE_Kokkos:BOOL=ON',
+                    '-DSEACASProj_ENABLE_OpenMP:Bool=ON',
+                    '-DKOKKOS_ENABLE_DEPRECATED_CODE:BOOL=OFF',
+                    '-DKokkos_ENABLE_Pthread:BOOL=OFF',
+                ])
+        else:
+            options.extend([
+                '-DSEACASProj_ENABLE_Kokkos:BOOL=OFF'
+            ])
+
+        if '+thread_safe' in spec:
+            options.extend([
+                '-DSEACASExodus_ENABLE_THREADSAFE:BOOL=ON',
+                '-DSEACASIoss_ENABLE_THREADSAFE:BOOL=ON',
+            ])
+
+        # ######################### TPLs #############################
+
+        # Note: -DXYZ_LIBRARY_NAMES= needs semicolon separated list of names
+        options.extend([
+            '-DTPL_ENABLE_Netcdf:BOOL=ON',
+            '-DNetCDF_ROOT:PATH=%s' % spec['netcdf'].prefix,
+            '-DTPL_ENABLE_X11:BOOL=%s' % (
+                'ON' if '+x11' in spec else 'OFF'),
+        ])
+
+        if '+hdf5' in spec:
+            options.extend([
+                '-DTPL_ENABLE_HDF5:BOOL=ON',
+                '-DHDF5_INCLUDE_DIRS:PATH=%s' % spec['hdf5'].prefix.include,
+                '-DHDF5_LIBRARY_DIRS:PATH=%s' % spec['hdf5'].prefix.lib
+            ])
+        else:
+            options.extend(['-DTPL_ENABLE_HDF5:BOOL=OFF'])
+
+        if '+metis' in spec:
+            options.extend([
+                '-DTPL_ENABLE_METIS:BOOL=ON',
+                '-DMETIS_LIBRARY_DIRS=%s' % spec['metis'].prefix.lib,
+                '-DMETIS_LIBRARY_NAMES=metis',
+                '-DTPL_METIS_INCLUDE_DIRS=%s' % spec['metis'].prefix.include,
+                '-DTPL_ENABLE_ParMETIS:BOOL=ON',
+                '-DParMETIS_LIBRARY_DIRS=%s;%s' % (
+                    spec['parmetis'].prefix.lib, spec['metis'].prefix.lib),
+                '-DParMETIS_LIBRARY_NAMES=parmetis;metis',
+                '-DTPL_ParMETIS_INCLUDE_DIRS=%s;%s' % (
+                    spec['parmetis'].prefix.include,
+                    spec['metis'].prefix.include)
+            ])
+        else:
+            options.extend([
+                '-DTPL_ENABLE_METIS:BOOL=OFF',
+                '-DTPL_ENABLE_ParMETIS:BOOL=OFF',
+            ])
+
+        if '+mpi' in spec:
+            options.extend([
+                '-DTPL_ENABLE_Pnetcdf:BOOL=ON',
+                '-DTPL_Netcdf_Enables_Netcdf4:BOOL=ON',
+                '-DTPL_Netcdf_PARALLEL:BOOL=ON',
+                '-DPNetCDF_ROOT:PATH=%s' % spec['parallel-netcdf'].prefix
```

Comment:
Missing a parallel-netcdf dependency.

Reply:
This should probably be removed and handled via the NetCDF variants.  I do specify that if `+mpi` is specified, then seacas depends on `netcdf@4.6.2+mpi+parallel-netcdf`